client/driver/qemu.go

```diff
@@ -233,13 +242,13 @@ func (d *QemuDriver) Start(ctx *ExecContext, task *structs.Task) (*StartResponse
 	if d.driverConfig.GracefulShutdown {
 		// This socket will be used to manage the virtual machine (for example,
 		// to perform graceful shutdowns)
-		monitorPath, err = getMonitorPath(ctx.TaskDir.Dir, ctx.TaskEnv.NodeAttrs[qemuDriverLongMonitorPathAttr])
-		if err == nil {
-			d.logger.Printf("[DEBUG] driver.qemu: got monitor path OK: %s", monitorPath)
-			args = append(args, "-monitor", fmt.Sprintf("unix:%s,server,nowait", monitorPath))
-		} else {
-			d.logger.Printf("[WARN] driver.qemu: %s", err)
+		monitorPath, err := d.getMonitorPath(ctx.TaskDir.Dir)
+		if err != nil {
+			d.logger.Printf("[ERR] driver.qemu: could not get qemu monitor path - error: %s", err)
```

Comment:
"[ERR] driver.qemu: could not get qemu monitor path: %s"

Reply:
Done!